src/models/assignments.ts

```diff
@@ -19,13 +19,15 @@ export class Assignment extends Model implements AssignmentType {
   title: string;
   points: number;
   due: string;
+  start: string;
```

Comment:
is this supposed to be the `timestamp` listed in the schema? Or is this a new, different field?

Reply:
yes, should be the same type of thing as due

Reply:
Let's change this to `timestamp` to be more consistent with the schema

Reply:
So it would be assignment.timestamp instead of assignment.start?

Reply:
yeah

Reply:
actually, nevermind. I was looking at the wrong schema

Reply:
And I saw you said something where start date was -2 weeks, I was trying to find that to replace it